### ISSUE TITLE
How to evaluate the performance?

### ISSUE THREAD

**Issue by Originlightwkp** [THUDM/CogVLM#510]
### System Info / 系統信息

cuda 12.1
pytorch 2.3.1

### Information / 问题信息

- [X] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

My test results are as follows, is this considered good or bad?

[2024-07-19 09:38:21,589] [INFO] [RANK 0]  validation loss at the end of training for test data | loss: 0.000000E+00                | PPL: 1.000000E+00 acc 5.319865E-02 | acc_w/o_case 5.319865E-02 |

### Expected behavior / 期待表现

None

**Comment by gyjlll** [comment]
Can you run evaluate_cogagent_demo.py correctly without reporting an error?
